app/scripts/controllers/specialErrorDetail.js

```diff
@@ -26,23 +28,48 @@ module.exports = /*@ngInject*/ function ($scope, $routeParams, $location, $http,
         $scope.error = {};
     }
 
+    $scope.sort = function(property) {
+        $scope.reverse = $scope.sortedBy === property ? !$scope.reverse : false;
```

Comment:
`$scope.reverse` seems like an odd variable name. What is this actually tracking?